modules/lead/sdm/operator-toolchain-values.tpl

```diff
@@ -45,6 +51,8 @@ operators:
       value: ${region}
   jenkins:
     enabled: ${operator_jenkins_enabled}
+    image:
+      repository: ${image_repository}
```

Comment:
How can these all share the same repository?  Does that mean the same docker image will be used for each of these?

Reply:
Also the product operator image and converge image are missing

Reply:
We could make the image repo the same value but I think long term we are going to break each of these into their own chart anyway.

Reply:
Oops, I misread the first comment but everything should be fixed